### PR TITLE
Readiness and liveness probe updates

### PR DESCRIPTION
**Pull request by DerekRoberts**
The frontend's probes were inconsistent, not using `/actuator/health` for OpenShift.  Oracle-api was mixed, using `/actuator/health` in some places and `/check` in others.  The rest seem fine.

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend](https://nr-spar-116-backend.apps.silver.devops.gov.bc.ca/)
[Frontend](https://nr-spar-116-frontend.apps.silver.devops.gov.bc.ca/)
[Oracle-API](https://nr-spar-116-oracle-api.apps.silver.devops.gov.bc.ca/)

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge-main.yml)

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend](https://nr-spar-116-backend.apps.silver.devops.gov.bc.ca/)
[Frontend](https://nr-spar-116-frontend.apps.silver.devops.gov.bc.ca/)
[Oracle-API](https://nr-spar-116-oracle-api.apps.silver.devops.gov.bc.ca/)

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge-main.yml)

### CODE REVIEW

File: frontend/openshift.deploy.yml [hunk 1]
```diff
@@ -136,7 +136,7 @@ objects:
                 successThreshold: 1
                 failureThreshold: 5
                 httpGet:
-                  path: /
+                  path: /actuator/health
```
Comment (RMCampos) [inline frontend/openshift.deploy.yml:139]: In the frontend we can keep this path `/`

**Review by RMCampos — CHANGES_REQUESTED** [review]
Good job. Just a minor issue in the frontend.

File: frontend/openshift.deploy.yml (hunk 1 above)
Comment (DerekRoberts) [reply frontend/openshift.deploy.yml:139]: Thanks and corrected!  :)

File: .github/workflows/merge-main.yml [hunk 2]
```diff
@@ -83,11 +83,12 @@ jobs:
           - name: backend
             file: backend/openshift.deploy.yml
             overwrite: true
+            verification_path: "actuator/health"
             parameters: -p BUILD=test
           - name: frontend
             file: frontend/openshift.deploy.yml
             overwrite: true
-            verification_path: "actuator/health"
+            verification_path: "/"
```
Comment (RMCampos) [inline .github/workflows/merge-main.yml:91]: 👍🏻 yep

File: .github/workflows/merge-main.yml [hunk 3]
```diff
@@ -99,7 +100,7 @@ jobs:
           - name: oracle-api
             file: oracle-api/openshift.deploy.yml
             overwrite: true
-            verification_path: "check"
+            verification_path: "actuator/health"
```
Comment (RMCampos) [inline .github/workflows/merge-main.yml:103]: 👍🏻  Yup. Working

File: .github/workflows/merge-main.yml [hunk 4]
```diff
@@ -192,11 +193,12 @@ jobs:
           - name: backend
             file: backend/openshift.deploy.yml
             overwrite: true
+            verification_path: "actuator/health"
             parameters: -p BUILD=prod
           - name: frontend
             file: frontend/openshift.deploy.yml
             overwrite: true
-            verification_path: "actuator/health"
+            verification_path: "/"
```
Comment (RMCampos) [inline .github/workflows/merge-main.yml:201]: All good here too.

**Review by RMCampos — APPROVED** [review]
LGTM. Although some checks are wrong. Sending a commit to get them fixed now.

**Review by craigyu — DISMISSED** [review]
⛑️

File: .github/workflows/pr-open.yml [hunk 5]
```diff
@@ -98,7 +98,7 @@ jobs:
           - name: frontend
             file: frontend/openshift.deploy.yml
             overwrite: true
-            verification_path: "actuator/health"
+            verification_path: "/"
```
Comment (RMCampos) [inline .github/workflows/pr-open.yml:101]: We can remove this sentence I believe. Since `/` is the default. :)

**Review by RMCampos — CHANGES_REQUESTED** [review]
@craigyu great job! 👍🏻 🥇

File: .github/workflows/pr-open.yml (hunk 5 above)
Comment (craigyu) [reply .github/workflows/pr-open.yml:101]: do I remove this in merge-main.yml as well?

Comment (RMCampos) [reply .github/workflows/pr-open.yml:101]: Yes please :)

Comment (craigyu) [reply .github/workflows/pr-open.yml]: [removed](https://github.com/bcgov/nr-spar/pull/116/commits/9963bbc7eec745743f5eac5d80f0e0d33113d620)

**Review by craigyu — APPROVED** [review]
Merging this in to see if it fixes deployment error

**Pull request merged**